instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/internal/LoggingEventMapper.java

```diff
@@ -221,8 +221,20 @@ private static void captureKeyValuePairAttributes(
     List<KeyValuePair> keyValuePairs = loggingEvent.getKeyValuePairs();
     if (keyValuePairs != null) {
       for (KeyValuePair keyValuePair : keyValuePairs) {
+        Object value = keyValuePair.value;
         if (keyValuePair.value != null) {
-          attributes.put(getAttributeKey(keyValuePair.key), keyValuePair.value.toString());
+          if (value instanceof Boolean) {
```

Comment:
Might it be more friendly for code reading by adding some comments explaining what types are processed now?

Reply:
I'm not sure what comment you are asking for.

Reply:
> I'm not sure what comment you are asking for.

I mean we can add some comment to describe what types are supported and what types are not supported to preserve attribute type now.